pkg/util/exec/exec.go

```diff
@@ -67,6 +70,43 @@ func (*CommandExecutor) ExecuteCommand(debug bool, actionName string, command st
 	return nil
 }
 
+// ExecuteCommandWithTimeout starts a process and wait for its completion with timeout.
+func (*CommandExecutor) ExecuteCommandWithTimeout(debug bool, timeout time.Duration, actionName string, command string, arg ...string) (string, error) {
```

Comment:
this function definitely needs a call to the `logCommand(debug, command, arg...)` func at the beginning like all the other `ExecuteCommand...` functions do.  The command being executed such as `rbd map ...` isn't being logged right now and that is super valuable information.

Reply:
sounds good.. I will add it